### ISSUE TITLE
[docs] Suggestion / Change Request

### ISSUE THREAD

**Issue by dixia** [EOSIO/welcome#199]
File: [docs/02_getting-started/03_smart-contract-development/10_payable_actions.md](https://github.com/EOSIO/welcome/blob/develop/docs/02_getting-started/03_smart-contract-development/10_payable_actions.md)

Suggest change the title to Payable Actions in Smart Contract

**Comment by bobgt** [comment]
Yes, **Payable Actions in Smart Contract** is a better and complete title.
